sickbeard/properFinder.py

```diff
@@ -265,34 +266,34 @@ def _downloadPropers(self, properList):
             historyResults = main_db_con.select(
                 "SELECT resource FROM history " +
                 "WHERE showid = ? AND season = ? AND episode = ? AND quality = ? AND date >= ? " +
-                "AND action IN (" + ",".join([str(x) for x in Quality.SNATCHED + Quality.DOWNLOADED]) + ")",
+                "AND (action LIKE '%2' OR action LIKE '%4')",
                 [curProper.indexerid, curProper.season, curProper.episode, curProper.quality,
                  historyLimit.strftime(History.date_format)])
 
             # make sure that none of the existing history downloads are the same proper we're trying to download
-            clean_proper_name = self._genericName(helpers.remove_non_release_groups(curProper.name))
+            clean_proper_name = self._genericName(remove_strings(remove_non_release_groups(curProper.name), ['.mkv','.avi','.mp4']))
```

Comment:
Again we are losing focus. If we keep overthinking all providers in two weeks we will have 50 PRs open and no bug fixes.

We dont need to sanitize else where. Only in propers. But if you have a better fix for the issue feel free to open another PR. This one worked so far.